web-server/sgx-wallet-test/app/codegen/Enclave_u.c

```diff
@@ -209,6 +209,14 @@ typedef struct ms_u_open64_ocall_t {
 	int ms_mode;
 } ms_u_open64_ocall_t;
 
+typedef struct ms_u_openat_ocall_t {
```

Comment:
Why are these structs getting added to `Encalve_u.c`?

Reply:
No idea.  It gets generated automatically by `bindgen`